TreeMaker/python/doZinvBkg.py

```diff
@@ -181,6 +181,11 @@ def doZinvBkg(process,tagname,geninfo,residual):
     process.TreeMaker2.VectorBool.append("goodPhotons:nonPrompt(photon_nonPrompt)")
     process.TreeMaker2.VectorBool.append("goodPhotons:fullID(photon_fullID)")
 
+    ## add MadGraph-level deltaR between photon or Z and partons
+    ## mainly used to make GJets and GJets0p4 MC samples orthogonal
+    process.minDeltaR = cms.EDProducer("MinDeltaRDouble")
+    process.TreeMaker2.VarsDouble.extend(['minDeltaR'])

```

Comment:
Can you give this branch a more descriptive name? "minDeltaR" is very generic.

At least "madMinDeltaR" would be good (our other MadGraph-based variable is madHT).